ocs-api/outils/outils.go

```diff
@@ -410,8 +417,17 @@ func RunCmd(commandString string, args ...string) ([]byte, []byte, error) {
 	// Now block waiting for the command to complete
 	err = cmd.Wait()
 	if err != nil {
-		return stdoutBytes, stderrBytes, errors.New("command " + commandString + " returned error: " + err.Error() + ". Stderr: " + string(stderrBytes))
+		if exitError, ok := err.(*exec.ExitError); ok {
+			codeOfExit := exitError.ExitCode()
+			s2 := strconv.Itoa(codeOfExit)
```

Comment:
Why convert `codeOfExit` to a string? It is an integer, which is what it really is. Remove this line and then change the `if` statement below to `if codeOfExit == 3 {`